src/main/scala/com/fulcrumgenomics/vcf/filtration/ReadEndSomaticVariantFilter.scala

```diff
@@ -113,37 +111,110 @@ class EndRepairArtifactLikelihoodFilter(val distance: Int = 2,
     // Apply a prior based on the MAF
     val totalObs = altGood + altBad + refGood + refBad
     val maf      = if (totalObs > 0) (altGood + altBad) / totalObs.toDouble else 0
-    val (priorMutation, priorArtifact) = priors(pileup, maf)
+    val (priorMutation, priorArtifact) = ReadEndSomaticVariantFilter.priors(pileup, maf)
     llMutation += LnProb.toLogProbability(priorMutation)
     llArtifact += LnProb.toLogProbability(priorArtifact)
 
     val total = LnProb.or(llMutation, llArtifact)
     val pMutation = LnProb.expProb(LnProb.normalizeByLogProbability(llMutation, total))
-    val pArtifact = LnProb.expProb(LnProb.normalizeByLogProbability(llArtifact, total))
-
     Map(Info.id -> pMutation)
   }
+}
+
+/** The companion object for [[ReadEndSomaticVariantFilter]]. */
+object ReadEndSomaticVariantFilter extends LazyLogging {
 
   /** Calculates a pair of priors:
-    *   The first is the prior that the genotype is the result of a true somatic mutations
-    *   The second is the prior that the genotype is the result of the end-repair artifact process
+    *   <priorMutation> is the prior that the genotype is the result of a true somatic mutation.
+    *   <priorArtifact> is the prior that the genotype is an artifact resulting from the mechanism specific to the filter
+    *
     *
     * Default implementation gives a mutation prior of {{{(maf * 2)^2}}} capped at 0.9999 and an
     * artifact prior of 1 - mutation prior.  This has the effect of creating priors that are heavily
     * skewed towards the artifact at very low MAFs, and towards mutations as the MAF increases towards 0.5.
     */
   private[filtration] def priors(pileup: Pileup[PileupEntry], maf: Double) : (Double, Double) = {
```

Comment:
I don't think this should be in the companion as implementations of the trait may want to override it (as suggested by `Default implementation...` in the docs.